govcd/disk.go

```diff
@@ -62,8 +63,12 @@ func (vdc *Vdc) CreateDisk(diskCreateParams *types.DiskCreateParams) (*Disk, err
 	}
 
 	// Send Request
-	req := vdc.client.NewRequest(nil, http.MethodPost, *reqUrl, bytes.NewBufferString(xml.Header+string(xmlPayload)))
-	req.Header.Add("Content-Type", execLink.Type)
+	reqPayload := bytes.NewBufferString(xml.Header + string(xmlPayload))
+	req := vdc.client.NewRequest(nil, http.MethodPost, *reqUrl, reqPayload)
+	req.Header.Add("Content-Type", createDiskLink.Type)
+	util.Logger.Printf("[DEBUG] POSTING TO URL: %s", reqUrl)
+	util.Logger.Printf("[DEBUG] XML TO SEND:\n%s", reqPayload)
```

Comment:
All information regarding request and response is logged automatically so no need explicitly log that. You can see implementation in logging.go